tor_core/helpers.py

```diff
@@ -84,22 +84,19 @@ def clean_list(items):
     return cleaned
 
 
-def send_to_slack(message, config, channel='#general'):
+def send_to_modchat(message, config, channel="general"):
```

Comment:
Am I completely blind, or are all references to `send_to_slack` not updated yet?

Also, using double quotes for strings is a crime :)

Reply:
https://github.com/GrafeasGroup/tor/pull/101 Did I miss any?

Also, yes. Need to fix the strings.

Reply:
Ah, yes I am blind. Forgot there are separate repos lol